mypy is not ignoring items specified in mypy-type-checker.ignorePatterns

After switching from my local mypy installation to the VSCode mypy extension, I am struggling with mypy failing to ignore the specified files/folders.

My `settings.json` contains the following:

```
"mypy-type-checker.args": [
    "--strict",
    "--implicit-reexport",
    "--namespace-packages",
    "--show-column-numbers",
    // "--cache-dir=.mypy_cache/.vscode",
],
"mypy-type-checker.preferDaemon": true,
"mypy-type-checker.reportingScope": "workspace",
"mypy-type-checker.ignorePatterns": [
    "**/graveyard/*.py",
    "**/graveyard/*.txt",
],
```

The `Mypy Type Checker` confirms this when outputting `[info] Global settings:`

```
"ignorePatterns": [
    "**/graveyard/*.py",
    "**/graveyard/*.txt",
],
```

However, it still outputs the following:

```
/mnt/c/Users/bread/Documents/abz_code/graveyard/simulator_old.py:244: error: Name "ctp.SimTemplate" is not defined  [name-defined]
```

I would appreciate any advice on what I am doing wrong and/or what mypy is doing wrong.
Thank you in advance!

--- a/bundled/tool/lsp_server.py
+++ b/bundled/tool/lsp_server.py
@@ -495,7 +495,7 @@ def _get_global_defaults():
                 "note": "Information",
             },
         ),
-        "ignorePatterns": [],
+        "ignorePatterns": GLOBAL_SETTINGS.get("ignorePatterns", []),
         "importStrategy": GLOBAL_SETTINGS.get("importStrategy", "useBundled"),
         "showNotifications": GLOBAL_SETTINGS.get("showNotifications", "off"),
         "extraPaths": GLOBAL_SETTINGS.get("extraPaths", []),

